domains/azumi.is-an.app.json

```diff
@@ -12,9 +12,9 @@
   },
   "nested": [
     {
-      "subdomain": "railway",
+      "subdomain": "docs",
       "record": {
-        "CNAME": "azumi.up.railway.app"
+        "CNAME": "azumi-development.github.io/docs"
```

Comment:
`azumi-development.github.io/docs` is not a domain name and cannot be used here

Reply:
yeah i know, what record can i use to do this though?

Reply:
None. You may be confusing DNS and routing

Reply:
DNS is for resolving IP addresses from domain names. It does not know anything about, URLs, HTTP, HTTPS, and so on. Just domain name => IP address, not more

Reply:
welp that's fine
but how do i set up email stuff and that, because i kinda need to get that workin

Reply:
aaaaaaaaaaa it doesn't support it